### DISCUSSION TITLE
看看我的导航页怎么样？

### DISCUSSION THREAD

**Discussion by docker-master** [hslr-s/sun-panel#72] [Show and tell]
![Cache_8220426c9cbbe41](https://github.com/hslr-s/sun-panel/assets/58912611/f3185bf1-6bd5-43f5-b75e-a892955ec863)
![microsoft_edge_screenshot_2024年1月21日 GMT+08_00 15_27_49 png](https://github.com/hslr-s/sun-panel/assets/58912611/2b5867ef-2124-4035-9b90-634ae83438bb)

**Comment by chen1204-st** [comment]
请问一下图标哪里找的

**Reply by docker-master** [reply]
群里面

**Reply by avsba001** [reply]
你的名字

背景图压缩到多大了啊
